artifacts/definitions/Windows/Detection/BinaryRename.yaml

```diff
@@ -25,43 +25,76 @@ parameters:
-        nbtstat.exe,nbtinfo.exe,nbtinfo.exe,Microsoft TCP/IP NetBios Information
-        nltest.exe,nltestrk.exe,nltestrk.exe,Microsoft® Logon Server Test Utility
-        qprocess.exe,qprocess,qprocess.exe,Query Process Utility
-        qwinsta.exe,qwinsta,qwinsta.exe,Query Session Utility
-        XXXXX,nc,nc.exe,NetCat for Windows - https://github.com/diegocr/netcat
-        XXXXX,AdFind.exe,AdFind.exe,Joeware ADFind
-        XXXXX,rclone,rclone.exe,Rsync for cloud storage
-        XXXXX,MEGAsync.exe,MEGAsync.exe,MEGAsync
-        XXXXX,MEGAcmdShell.exe,MEGAcmdShell,MEGAcmdShell
-        XXXXX,pCloud.exe,pCloud.exe,pCloud cloud storage
-        XXXXX,,pCloud Drive.exe,pCloud setup
-        XXXXX,mimikatz,mimikatz.exe,Top 5 tool: Mimikatz
-        procdump.exe,ProcDump,procdump,Sysinternals process dump utility
-        procdump64.exe,ProcDump,procdump,Sysinternals process dump utility
+        Filename,Internal,Original,Description,Note
+        cmd.exe,cmd,Cmd.Exe,Windows Command Processor,cmd.exe
+        7z.exe,7z,7z.exe,7-Zip Console,7z.exe
+        certutil.exe,CertUtil.exe,CertUtil.exe,CertUtil,certutil.exe
+        cmstp.exe,CMSTP,CMSTP.EXE,Microsoft Connection Manager Profile Installer,cmstp.exe
+        cscript.exe,cscript.exe,cscript.exe,Microsoft ® Console Based Script Host,cscript.exe
+        mshta.exe,MSHTA.EXE,MSHTA.EXE,Microsoft ® HTML Application host,mshta.exe
+        msiexec.exe,msiexec,msiexec.exe,Windows® installer,msiexec.exe
+        powershell.exe,POWERSHELL,PowerShell.EXE,Windows PowerShell,powershell.exe
+        psexec.exe,PsExec,psexec.c,Sysinternals PSExec,psexec.exe
+        psexec64.exe,PsExec,psexec.exe,Sysinternals PSExec,psexec64.exe
+        regsvr32.exe,REGSVR32,REGSVR32.EXE,Microsoft© Register Server,regsvr32.exe
+        rundll32.exe,rundll,RUNDLL32.EXE,Windows host process (Rundll32),rundll32.exe
+        winrar.exe,WinRAR,WinRAR.exe,WinRAR archiver,winrar.exe
+        wmic.exe,wmic.exe,wmic.exe,WMI Commandline Utility,wmic.exe
+        wscript.exe,wscript.exe,wscript.exe,Microsoft ® Windows Based Script Host,wscript.exe
+        wevtutil.exe,wevtutil.exe,wevtutil.exe,,wevtutil.exe
+        net.exe,net.exe,net.exe,,net.exe
+        net1.exe,net1.exe,net1.exe,,net1.exe
+        netsh.exe,netsh.exe,netsh.exe,,netsh.exe
+        powershell_ise.exe,powershell_ise.exe,powershell_ise.exe,,powershell_ise.exe
+        dsquery.exe,dsquery.exe,dsquery.exe,Microsoft AD DS/LDS query command line utility,dsquery.exe
+        nbtstat.exe,nbtinfo.exe,nbtinfo.exe,Microsoft TCP/IP NetBios Information,nbtstat.exe
+        nltest.exe,nltestrk.exe,nltestrk.exe,Microsoft® Logon Server Test Utility,nltest.exe
+        qprocess.exe,qprocess,qprocess.exe,Query Process Utility,qprocess.exe
+        qwinsta.exe,qwinsta,qwinsta.exe,Query Session Utility,qwinsta.exe
+        XXXXX,nc,nc.exe,NetCat for Windows - https://github.com/diegocr/netcat,nc.exe
+        XXXXX,AdFind.exe,AdFind.exe,Joeware ADFind,AdFind.exe
```

Comment:
Maybe clearer to use * here instead?

Reply:
does that work as you expect? The name can be any string as XXXX entries are just looking for the orginal and internal names.

Reply:
Yeah I saw that but XXXX is meaningless but * implies anything.

Maybe use ANY or something?